bindings/c/src/common.rs

```diff
@@ -1080,9 +1085,9 @@ impl ActivationHandler for FfiActivationHandler {
 }
 
 type ActionHandlerCallbackUnwrapped =
-    extern "C" fn(request: *const action_request, userdata: *mut c_void);
+    extern "C" fn(request: *mut action_request, userdata: *mut c_void);
 pub type ActionHandlerCallback =
```

Comment:
I think we should document the ownership transfer here and mention `accesskit_action_handler_free`.